docs/how-to-customize-my-service/setting-up-a-custom-sftp-server-name.md

```diff
@@ -27,3 +28,6 @@ Here are some links to the documentation regarding editing DNS records with popu
 | Wordpress | [Adding Custom DNS Records](https://wordpress.com/support/domains/custom-dns/#adding-custom-dns-records) |
 | Wix | [Adding or Updating CNAME Records](https://support.wix.com/en/article/adding-or-updating-cname-records-in-your-wix-account) |
 
+:::tip
+Adding a custom domain for your SFTP endpoint is available in all plans. However, if you'd like to change the domain that shows as default in SFTP To Go's admin dashboard, read more about this option [here](../getting-started/organization-settings#domains). This feature is only available with [certain plans](https://sftptogo.com/pricing).
```

Comment:
@saggineumann The ability to set a custom domain for SFTP / FTPS endpoints through the Domains settings is available for all plans. Only web portal domains is restricted for certain plans.

Reply:
good to know! :) i'll fix it.